mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx

```diff
@@ -194,6 +194,83 @@ export function AnnouncementBanner ({
   )
 }
 
+export function AnnouncementBannerV2 ({
+  hideAnnouncement,
+  announcement,
+  testID
+}: AnnouncementBannerProps): JSX.Element {
+  const { isLight } = useThemeContext()
+  const isOtherAnnouncement = announcement.type === undefined || announcement.type === 'OTHER_ANNOUNCEMENT'
+  return (
+    <ThemedView
+      testID={testID}
+      style={tailwind('mt-9 relative mx-5 px-5 py-3 flex flex-row items-center border-0.5 rounded-xl-v2', {
```

Comment:
Can move `mt-9` and `mx-5` in Announcements.tsx by wrapping it in a View component? Composite screen also reuses this

Reply:
let's revisit when we work on dex 2.0